app/assets/stylesheets/lifehack_style.scss

```diff
@@ -160,9 +188,70 @@
 }
 
 .add-lifehack {
-  border: 1px solid $green;
+  border: 0.1rem solid $oil;
   border-radius: 28px;
-  display: inline-block;
   font-size: 15px;
-  padding: 12px 20px;
+  width: 500px;
+  font-size: 26px;
+  background-color: $turquoise;
+}
+
+.top-action-bar {
+  text-align: center;
+  padding-bottom: 15px;
+}
+
+.review {
+
+  table {
+    border-bottom: 2px solid $oil;
+    margin-top: 20px;
+    background-color: $cornsilk;
+  }
+  th {
+    text-align: center;
+    padding:1px 71px 0px 70px;
+  }
+
+  td {
+    text-align: center;
+      padding: 0 0 0 0;
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Shorthand form for property `padding` should be written more concisely as `0` instead of `0 0 0 0`